src/Sentry.Unity/Integrations/UnitySocketExceptionFilter.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using Sentry.Extensibility;
+
+namespace Sentry.Unity.Integrations
+{
+    internal class UnitySocketExceptionFilter : IExceptionFilter
+    {
+        internal const string Message = "The requested address is not valid in this context";
```

Comment:
Since these are human-readable messages, I wonder if they are ever localized to non-English languages?  Does Unity change these messages if the Editor is configured for some other language, or perhaps if the `CultureInfo.CurrentUICulture` is not English?